include/exadg/incompressible_navier_stokes/postprocessor/postprocessor.cpp

```diff
@@ -160,6 +161,14 @@ PostProcessor<dim, Number>::do_postprocessing(VectorType const &     velocity,
       velocity_magnitude.evaluate(velocity);
       additional_fields_vtu.push_back(&velocity_magnitude);
     }
+    if(pp_data.output_data.write_wall_shear_stress_on_IDs.size() > 0)
+    {
+      wall_shear_stress.evaluate(velocity);
+      if(dim == 3)
+        surface_fields_vtu.push_back(&wall_shear_stress);
+      else
+        additional_fields_vtu.push_back(&wall_shear_stress);
```

Comment:
Right now, I do not understand this. Does it mean that a volume field is written in 2D for the wall shear stress? This would not be intuitive and require some explanation in my opinion.

Reply:
Yes, unfortunately. deal.II does not allow output of 2D `SurfaceOutput`. Hence I write a volumetric data. And yes, I agree that this is ugly =(